Tests/NIOSSLTests/TLSConfigurationTest.swift

```diff
@@ -254,6 +254,15 @@ class TLSConfigurationTest: XCTestCase {
         return try assertHandshakeSucceededEventLoop(withClientConfig: clientConfig, andServerConfig: serverConfig, file: file, line: line)
         #endif
     }
+    
+    func setupTLSLeafandClientIdentitiesFromCustomCARoot() throws -> (NIOSSLCertificate, NIOSSLPrivateKey , NIOSSLCertificate, NIOSSLPrivateKey) {
```

Comment:
As a minor test cleanup, let's use labels for these fields in the tuple so that we can access them by name in the test cases instead of as `.0` and `.1` and so-on. Additionally, in the test cases can you give this tuple a camelCase name?

Reply:
@Lukasa this has been addressed in the latest commit.